client/asset/btc/spv.go

```diff
@@ -1036,13 +1070,13 @@ func (w *spvWallet) startWallet() error {
 	// simnet harness alpha node as an additional peer so we don't have to type
 	// it in.
 	if w.chainParams.Name == "regtest" && len(w.connectPeers) == 0 {
-		w.connectPeers = append(w.connectPeers, "localhost:20575")
+		w.connectPeers = []string{"localhost:20575"}
 	}
-
 	chainService, err := neutrino.NewChainService(neutrino.Config{
-		DataDir:          w.netDir,
-		Database:         w.neutrinoDB,
-		ChainParams:      *w.chainParams,
+		DataDir:     w.netDir,
+		Database:    w.neutrinoDB,
+		ChainParams: *w.chainParams,
+		// AddPeers:         []string{"localhost"}, // worth a shot
```

Comment:
Almost 4 minutes exactly with a remote mainnet peer.  This can be painful if you get a poor peer, and neutrino doesn't do multipeer, at least not for getcfilters, so I put up a node at 45.76.69.19 and got the 4 minute sync result.  I think we can hard code this as an addpeer to make it less likely a user will get a terrible sync nodes for their compact filters.